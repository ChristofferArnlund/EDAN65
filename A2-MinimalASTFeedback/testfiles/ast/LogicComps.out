Program
  List
    Func
      Name ID="a"
      List
      List
        IdDecl
          Name ID="x"
          Opt
        IdDecl
          Name ID="y"
          Opt
        IdUse
          Name ID="x"
          Ge
            Numeral NUMERAL="5"
            Add
              Mul
                Numeral NUMERAL="3"
                Numeral NUMERAL="2"
              Numeral NUMERAL="4"
        IdUse
          Name ID="y"
          Not
            Add
              Numeral NUMERAL="4"
              Mul
                Numeral NUMERAL="3"
                Numeral NUMERAL="2"
            Numeral NUMERAL="1"
