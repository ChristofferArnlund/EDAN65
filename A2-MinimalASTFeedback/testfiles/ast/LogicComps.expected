Program
  List
    Func
      IdDecl
        Name ID="a"
        Opt
      List
      StmtBlock
        List
          IdDecl
            Name ID="x"
            Opt
          IdDecl
            Name ID="y"
            Opt
          Assign
            IdUse
              Name ID="x"
            Ge
              Numeral NUMERAL="5"
              Add
                Mul
                  Numeral NUMERAL="3"
                  Numeral NUMERAL="2"
                Numeral NUMERAL="4"
          Assign
            IdUse
              Name ID="y"
            Not
              Add
                Numeral NUMERAL="4"
                Mul
                  Numeral NUMERAL="3"
                  Numeral NUMERAL="2"
              Numeral NUMERAL="1"
